Android.mk

```diff
@@ -276,7 +276,15 @@ else
 $(warning Skipping build of hybris-updater-script since HYBRIS_BOOT_PART is not specified)
 endif
 
-HYBRIS_COMMON_ANDROID8_TARGETS := verity_signer boot_signer e2fsdroid vendorimage ramdisk libselinux_stubs libsurfaceflinger libsf_compat_layer libhwc2_compat_layer bootctl
+HYBRIS_COMMON_ANDROID8_TARGETS := e2fsdroid ramdisk libselinux_stubs libsurfaceflinger libsf_compat_layer libhwc2_compat_layer
+# Legacy-partitioned devices have no need to build slotselect or vendorimage
+ifeq ($(BOARD_PROPERTY_OVERRIDES_SPLIT_ENABLED),true)
+	HYBRIS_COMMON_ANDROID8_TARGETS += vendorimage bootctl
+endif
+# Generic test if dm-verity is enabled
+ifneq ($(PRODUCT_SYSTEM_VERITY_PARTITION),)
+	HYBRIS_COMMON_ANDROID8_TARGETS += verity_signer boot_signer
```

Comment:
Sony Xperia XA2 has `PRODUCT_SYSTEM_VERITY_PARTITION`, but we do not build `verity_signer` and `boot_signer`, because we have verity disabled there for Sailfish OS.
I need to test, and unsure when would have time for such, but would there be a certain pollution of the rootfs? cc @krnlyng

Reply:
I'm happy to drop the verity stuff altogether, I just didn't want to change the existing target list.

Reply:
Sorry my bad, I thought you introduced the two. Hence no regressions here.